docarray/array/stacked/list_advance_indexing.py

```diff
@@ -0,0 +1,42 @@
+from typing import Iterator, MutableSequence, TypeVar
+
+from docarray.array.array.sequence_indexing_mixin import IndexingSequenceMixin
+
+T_item = TypeVar('T_item')
+
+
+class ListAdvanceIndex(IndexingSequenceMixin[T_item]):
```

Comment:
```suggestion
class ListAdvancedIndexing(IndexingSequenceMixin[T_item]):
```
The `d` is needed I think, on the `ing` I could be convinced otherwise ;)